test/SmokeTests/OOProcSmokeTests/durableJava/extensions.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Check to see if changing the `TargetFramework` value to `net6.0` will resolve the CI problem. The exception you see in the failure message indicates that the wrong flavor of the extension was loaded - i.e., the version which isn't compatible with the changes requried for Java to work.